upp-factset-provisioner/README.md

```diff
@@ -0,0 +1,54 @@
+# UPP Factset RDS Provisioning
+
+* Provisioning script is currently only able to spin up aurora rds in eu-west-1
+
+## Building the Docker image
+
+The UPP Factset Provisioner can be built locally as a Docker image:
+
+`docker build -t upp-factset-provisioner:local .`
+
+## Provisioning a cluster
+
+The provisioning process will:
+
+* Creates an aurora cluster, db subnet group and cluster parameter group in eu-west-1 using the cloud formation template
+
+How to run:
+
+# Parameters
```

Comment:
I think the TAG_ENVIRONMENT should be included here (i.e. `t` or `p`)

Reply:
I originally had it as a parameter to be passed in but instead opted to include it in the content-test and content-prod ansible files as each account will only create t or p instances

Reply:
Ohh is it in the ansible vault files?

Reply:
The LastPass note doesn't contain the vault pass, can you add to that?

Reply:
I havent actually made a folder yet and almost went on holiday thanks :)